<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Header</class>
 <widget class="QWidget" name="Header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>32</number>
   </property>
   <property name="leftMargin">
    <number>64</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>64</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="pictureLabel">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <family>Corbel</family>
      </font>
     </property>
     <property name="text">
      <string>DILLON METZLER</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="homeButton">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="aboutButton">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>ABOUT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="projectsButton">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>PROJECTS</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
